kafka-rest/src/main/java/io/confluent/kafkarest/response/FakeAsyncResponse.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest.response;
+
+import java.util.Collection;
+import java.util.Date;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.GuardedBy;
+import javax.ws.rs.container.AsyncResponse;
+import javax.ws.rs.container.TimeoutHandler;
+
+/**
+ * A fake {@link AsyncResponse} to be used in tests.
```

Comment:
if it's only for use in tests, does it belong under `/main`?

Reply:
There's a difference between code which is used by tests, and tests. I'm used to keep code that is used by tests on main. That kind of forces me to do it properly and to test it. Keeping it as is.